samples/02_commands_framework/Modules/PublicModule.cs

```diff
@@ -49,6 +50,29 @@ public async Task BanUserAsync(IGuildUser user, [Remainder] string reason = null
             await ReplyAsync("ok!");
         }
 
+        // Kick a user
+        [Command("kick")]
```

Comment:
Adding only one sample command does not seem to really contribute to the existing project in a helpful manner; perhaps add more or something more helpful?